packages/components/src/Loader/Loader.snapshot.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Loader from './Loader.component';
+import { CIRCULAR_PROGRESS_SIZE as SIZE } from '../constants';
+
+jest.mock('react-dom');
+
+
+describe('Loader', () => {
+	it('should render a loader', () => {
+		const wrapper = renderer.create(<Loader />).toJSON();
+
+		expect(wrapper).toMatchSnapshot();
+	});
+
+	it('should render a small loader', () => {
```

Comment:
Your small and large tests produce the same snapshot as default :/
It's because of css modules, the classname is set to undefined.

We should modify the css module mock. Let's keep them so that we have tests when we change that.